From dd33cd29afc6dc5e42ee6d766d1825b089a3b932 Mon Sep 17 00:00:00 2001
From: David Andrey <david.andrey@netmodule.com>
Date: Wed, 28 Nov 2012 17:31:09 +0100
Subject: [PATCH 06/17] ARM: xilinx: Micrel PHY: disabled PAUSE

No Gbit link can be auto-negotiated with PAUSE
sym and asym enabled

Signed-off-by:David Andrey david.andrey@netmodule.com
---
 drivers/net/phy/micrel.c |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/drivers/net/phy/micrel.c b/drivers/net/phy/micrel.c
index 560241a..99d0e33 100644
--- a/drivers/net/phy/micrel.c
+++ b/drivers/net/phy/micrel.c
@@ -191,8 +191,7 @@ static struct phy_driver ksz9031_driver = {
 	.phy_id		= PHY_ID_KSZ9031,
 	.phy_id_mask	= 0x000ffffe,
 	.name		= "Micrel KSZ9031 Gigabit PHY",
-	.features	= (PHY_GBIT_FEATURES | SUPPORTED_Pause
-				| SUPPORTED_Asym_Pause),
+	.features	= PHY_GBIT_FEATURES, /* @NetModule, da, PAUSE disabled, else Gbit not working */
 	.flags		= PHY_HAS_MAGICANEG | PHY_HAS_INTERRUPT,
 	.config_init	= kszphy_config_init,
 	.config_aneg	= genphy_config_aneg,
-- 
1.7.4.1

